photon-ml/src/integTest/scala/com/linkedin/photon/ml/cli/game/scoring/DriverTest.scala

```diff
@@ -185,15 +185,39 @@ class DriverTest extends SparkTestUtils with TestTemplateWithTmpDir {
     val random = new Random(MathConst.RANDOM_SEED).self
     val scores = new KeyValueScore(sc.parallelize((0 until numSamples).map(idx => (idx.toLong, random.nextDouble()))))
     val labels = sc.parallelize((0 until numSamples).map(idx => (idx.toLong, random.nextInt(2))))
-    val gameDataSet = labels.mapValues(label =>
+
+    // Ensure that each queryId and documentId has both positive and negative labels, so that AUC will not return NaN
+    val staticGameData = Seq((0, 1), (1, 0)).flatMap { case (queryId, documentId) =>
+      Seq(0, 1).map { label =>
+        new GameDatum(
+          response = label,
+          offsetOpt = Some(0.0),
+          weightOpt = Some(1.0),
+          featureShardContainer = Map(),
+          idTypeToValueMap = Map(
+            "queryId" -> queryId.toString,
+            "documentId" -> documentId.toString)
+        )
+      }
+    }
+
+    // Add ids and parallelize
+    val staticGameDataSet = sc.parallelize(
+      (numSamples until numSamples + staticGameData.length)
```

Comment:
Nope - looks good - I was thrown off for a second thinking this was a loop, but it is a Range.